sass/layout/_markdown.scss

```diff
@@ -266,7 +266,7 @@ h6 {
         background: $white;
 
         &:nth-child(2n) {
-            background-color: #f8f8f8;
+            background: rgba(var(--center-channel-color-rgb), 0.07);
         }
```

Comment:
Unfortunately the CSS that sets the table background to white by default has more specificity than this update and so is overriding this CSS. 🙁

Here is the CSS that is now overriding:
```css
.app__body .markdown__table, .app__body .markdown__table tbody tr {
    background: rgb(255, 255, 255);
}
```

As you can see, the overriding CSS starts with the `.app__body` class, which the default CSS does not have as so overriding CSS taking precedence.

To fix this, we can use SASS's parent selector to wrap the background css statement like so:

```sass
&:nth-child(2n) {
    .app__body & {
        background: rgba(var(--center-channel-color-rgb), 0.07);
    }
}
```
... thereby adding `.app__body` to the beginning off this CSS selector statement so it looks like this:

```css
.app__body .markdown__table tbody tr:nth-child(2n) {
    background: rgba(var(--center-channel-color-rgb), 0.07);
}
```
Let me know if you have any questions!